### PR TITLE
Add mask2whole_mask and whole_mask2mask

### PR DESCRIPTION
**Pull request by yuyu2172**
```python
from fcis.datasets.coco_instance_segmentation_dataset import COCOInstanceSegmentationDataset
from fcis.datasets.coco_utils import coco_instance_segmentation_label_names
from fcis.utils import visualize_mask
import matplotlib.pyplot as plt
dataset = COCOInstanceSegmentationDataset(split='val')
img, bbox, label, mask = dataset[0]
whole_mask = mask2whole_mask(mask, bbox, img.shape[1:])
mask = [m.astype(np.float32) for m in mask]

img = img.transpose(1, 2, 0)
visualize_mask(img, label, mask, bbox, np.ones(len(bbox)), coco_instance_segmentation_label_names, 0.5)
plt.show()

plt.imshow(np.max(whole_mask, axis=0))
plt.show()
```

mask
figure_1-1

whole_mask
figure_1-2

### CODE REVIEW

**Comment by yuyu2172** [comment]
I found it really important to round bounding box coordinates instead of using `astype`.
`astype` always round off decimals.

File: fcis/datasets/coco_instance_segmentation_dataset.py [hunk 1]
```diff
@@ -84,12 +85,9 @@ def _get_annotations(self, i):
         label = np.array([self.cat_ids.index(ann['category_id'])
                           for ann in annotation], dtype=np.int32)
 
-        mask = list()
-        for anno, bb in zip(annotation, bbox):
-            m = self._segm_to_mask(anno['segmentation'], (H, W))
-            bb = bb.astype(np.int32)
-            m = m[bb[0]:bb[2], bb[1]:bb[3]]
-            mask.append(m)
+        whole_mask = [self._segm_to_mask(anno['segmentation'], (H, W))
```
Comment (knorth55) [inline fcis/datasets/coco_instance_segmentation_dataset.py]: `whole_mask` should be numpy array, isn't it?

**Comment by knorth55** [comment]
Can you squach and clean up commits?
Or can I squash?

**Comment by yuyu2172** [comment]
Yeah. Squash it.

**Comment by knorth55** [comment]
thx!

**Pull request merged**